seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -3602,12 +3602,12 @@
       "id": "56533eb9ac21ba0edf2244e0",
       "title": "Replacing If Else Chains with Switch",
```

Comment:
Umn, let's leave this one IMO. Challenge renaming is just a minor tweak, 

> if else if

But that will introduce tons of issues, broken links on porfolios, its better we leave it as it is.